scripts/generate_proto_stubs.sh

```diff
@@ -73,3 +73,5 @@ protoc_install/bin/protoc --proto_path=$PYTHON_PROTOBUF_DIR/src --mypy_out=$REPO
 
 isort $REPO_ROOT/stubs/protobuf
 black $REPO_ROOT/stubs/protobuf
+
+sed -i="" 's/mypy-protobuf [^"]*/mypy-protobuf 2.8/' $REPO_ROOT/stubs/protobuf/METADATA.toml
```

Comment:
Maybe just put `${MYPY_PROTOBUF_VERSION}` here?

Reply:
oops! Thanks for catching

Reply:
You need to switch to double quotes, because variables aren't substituted inside single quotes. But then you need to escape the double quote in the regex:

```suggestion
sed -i="" "s/mypy-protobuf [^\"]*/mypy-protobuf 2.8/" $REPO_ROOT/stubs/protobuf/METADATA.toml
```

Reply:
Yep @Akuli is right - I had to make that exact change.